### PR TITLE
Test v2 and v3 keys

### PR DESCRIPTION
**Pull request by liggitt**

### CODE REVIEW

File: roles/etcd_migrate/tasks/migrate.yml [hunk 1]
```diff
@@ -3,6 +3,27 @@
...
     l_etcd_service: "{{ 'etcd_container' if openshift.common.is_containerized else 'etcd' }}"
 
+- name: Enable etcd members
+  service:
+    name: "{{ l_etcd_service }}"
+    state: started
+
+- name: Pre-migrate key dump
+  shell: >
+    etcdctl
```
Comment (ingvagabund) [inline roles/etcd_migrate/tasks/migrate.yml]: please, remove the whitespace at the end of the line

File: roles/etcd_migrate/tasks/migrate.yml [hunk 2]
```diff
@@ -3,6 +3,27 @@
...
+- name: Enable etcd members
+  service:
+    name: "{{ l_etcd_service }}"
+    state: started
+
+- name: Pre-migrate key dump
+  shell: >
+    etcdctl 
+    --endpoints https://{{ etcd_peer }}:{{ etcd_client_port }}
+    --cert-file {{ etcd_peer_cert_file }}
```
Comment (ingvagabund) [inline roles/etcd_migrate/tasks/migrate.yml]: please, remove the whitespace at the end of the line

File: roles/etcd_migrate/tasks/migrate.yml [hunk 3]
```diff
@@ -3,6 +3,27 @@
...
+  service:
+    name: "{{ l_etcd_service }}"
+    state: started
+
+- name: Pre-migrate key dump
+  shell: >
+    etcdctl 
+    --endpoints https://{{ etcd_peer }}:{{ etcd_client_port }}
+    --cert-file {{ etcd_peer_cert_file }} 
+    --key-file {{ etcd_peer_key_file }}
```
Comment (ingvagabund) [inline roles/etcd_migrate/tasks/migrate.yml]: please, remove the whitespace at the end of the line

File: roles/etcd_migrate/tasks/migrate.yml [hunk 4]
```diff
@@ -3,6 +3,27 @@
...
+    name: "{{ l_etcd_service }}"
+    state: started
+
+- name: Pre-migrate key dump
+  shell: >
+    etcdctl 
+    --endpoints https://{{ etcd_peer }}:{{ etcd_client_port }}
+    --cert-file {{ etcd_peer_cert_file }} 
+    --key-file {{ etcd_peer_key_file }} 
+    --ca-file {{ etcd_peer_ca_file }}
```
Comment (ingvagabund) [inline roles/etcd_migrate/tasks/migrate.yml]: please, remove the whitespace at the end of the line

File: roles/etcd_migrate/tasks/migrate.yml [hunk 5]
```diff
@@ -33,13 +54,29 @@
...
+    etcdctl --cert-file {{ etcd_peer_cert_file }} --key-file {{ etcd_peer_key_file }} --ca-file {{ etcd_peer_ca_file }} --endpoints https://{{ etcd_peer }}:{{ etcd_client_port }} cluster-health
   register: l_etcd_migrate_health
   until: l_etcd_migrate_health.rc == 0
   retries: 3
   delay: 30
   run_once: true
 
+- name: Post-migrate key dump
+  shell: >
+    etcdctl
```
Comment (ingvagabund) [inline roles/etcd_migrate/tasks/migrate.yml]: please, remove the whitespace at the end of the line

File: roles/etcd_migrate/tasks/migrate.yml [hunk 6]
```diff
@@ -33,13 +54,29 @@
...
   until: l_etcd_migrate_health.rc == 0
   retries: 3
   delay: 30
   run_once: true
 
+- name: Post-migrate key dump
+  shell: >
+    etcdctl 
+    --endpoints https://{{ etcd_peer }}:{{ etcd_client_port }}
+    --cert-file {{ etcd_peer_cert_file }}
```
Comment (ingvagabund) [inline roles/etcd_migrate/tasks/migrate.yml]: please, remove the whitespace at the end of the line

File: roles/etcd_migrate/tasks/migrate.yml [hunk 7]
```diff
@@ -33,13 +54,29 @@
...
   retries: 3
   delay: 30
   run_once: true
 
+- name: Post-migrate key dump
+  shell: >
+    etcdctl 
+    --endpoints https://{{ etcd_peer }}:{{ etcd_client_port }}
+    --cert-file {{ etcd_peer_cert_file }} 
+    --key-file {{ etcd_peer_key_file }}
```
Comment (ingvagabund) [inline roles/etcd_migrate/tasks/migrate.yml]: please, remove the whitespace at the end of the line

File: roles/etcd_migrate/tasks/migrate.yml [hunk 8]
```diff
@@ -33,13 +54,29 @@
...
   delay: 30
   run_once: true
 
+- name: Post-migrate key dump
+  shell: >
+    etcdctl 
+    --endpoints https://{{ etcd_peer }}:{{ etcd_client_port }}
+    --cert-file {{ etcd_peer_cert_file }} 
+    --key-file {{ etcd_peer_key_file }} 
+    --ca-file {{ etcd_peer_ca_file }}
```
Comment (ingvagabund) [inline roles/etcd_migrate/tasks/migrate.yml]: please, remove the whitespace at the end of the line

**Comment by sdodson** [comment]
This was mostly meant for debugging of https://bugzilla.redhat.com/show_bug.cgi?id=1475351 but I have cleaned it up a bit once I got it working in my fork. Lets wait to see what we find out today and we'll decide if we should clean up and merge something like this.

**Pull request closed without merging**